python/shotgun_model/shotgun_model.py

```diff
@@ -440,6 +440,7 @@ def _load_data(
                                           each column in the row contains the value for the corresponding field from columns. This means
                                           that the data from the loaded entity will be available field by field. Subclasses can modify
                                           this behavior by overriding _get_additional_columns.
+        :param editable_columns:          A subset of ``columns`` that will be editable in views that use this model.
```

Comment:
Any reason to restrict it to a subset rather than just having the full column list be the union?

Otherwise there should be a check to make sure it is a subset since I think passing in invalid data will result in a kinda unintuitive error.